checkIfSorted.js

```diff
@@ -0,0 +1,13 @@
+function checkIfSorted(array) {
+  if (array instanceof Array) {
```

Comment:
Don't wrap whole code by checking data type. Use smth like this: if (! (array instanceof Array)) { return null };

Reply:
Done.